lib/miq_automation_engine/engine/miq_ae_engine.rb

```diff
@@ -67,45 +68,67 @@ def self.deliver_queue(args, options = {})
     options[:ae_state_previous] = YAML.dump(workspace.current_state_info) unless workspace.current_state_info.empty?
   end
 
+  def self.get_request_id(options)
+    # binding.pry
+    if options[:object_type].ends_with?("Request")
+      return options[:object_id]
+    end
+    if options[:object_type].ends_with?("Task")
+      return MiqRequestTask.find(options[:object_id]).miq_request_id
+    end
+    if options[:object_type].starts_with?("Service")
+      return Service.find(264).miq_request.id
```

Comment:
This is hardcoding a specific service, so will need to be fixed before completion.